src/main/java/org/gestern/gringotts/dependency/Dependency.java

```diff
@@ -35,7 +35,7 @@
     private Dependency() {
         factions = FactionsHandler.getFactionsHandler(hookPlugin(
                 "Factions",
-                "com.massivecraft.factions.Factions",
+                "com.massivecraft.factions.TerritoryAccess",
```

Comment:
Is this class reasonably stable between Factions versions?

Reply:
No idea. I just needed one that has been there a while (to not break backwards compatibility) but different to a class found in Savage factions which unfortunately uses the same group id.